stale

```diff
@@ -73,6 +73,18 @@ list_port() {
 		| jq -r '.[] | "\(.ID) \(."Updated At") \(.Name)"'
 }
 
+list_security_group() {
+	for resource_id in $(openstack security group list -f value -c ID); do
+		res="$(openstack security group show -f json -c updated_at -c name "$resource_id")"
+		update_time="$(jq -r '.updated_at' <<< "$res")"
+		name="$(jq -r '.name' <<< "$res")"
+		# we can't and don't want to remove the default security group
+		if [[ ${name} != "default" ]]; then
```

Comment:
Have you shellchecked your change locally? I am not sure that failing to wrap in double quotes won’t create problems

Reply:
(the presubmit only runs shellcheck on clean-ci-resources.sh unfortunately)